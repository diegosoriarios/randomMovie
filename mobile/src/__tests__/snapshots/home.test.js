// __tests__/Intro-test.js
import React from 'react';

import renderer from 'react-test-renderer';
import {useNavigation} from '@react-navigation/native';
import HomePage from '../../pages/home/index';

jest.mock('@react-navigation/native');

test('renders correctly', () => {
  const tree = renderer.create(<HomePage />).toJSON();
  expect(tree).toMatchSnapshot();
});
