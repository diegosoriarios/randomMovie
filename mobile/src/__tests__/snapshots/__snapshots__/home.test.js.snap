// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<View
  style={
    Object {
      "alignItems": "center",
      "backgroundColor": "white",
      "flex": 1,
      "justifyContent": "center",
    }
  }
>
  <View
    style={
      Object {
        "backgroundColor": "#00b4d8",
        "borderBottomLeftRadius": 50,
        "height": "70%",
        "left": "40%",
        "position": "absolute",
        "top": 0,
        "width": "70%",
      }
    }
  />
  <View
    style={
      Object {
        "flexDirection": "row",
        "justifyContent": "space-around",
        "position": "absolute",
        "top": 25,
        "width": "100%",
      }
    }
  >
    <View
      accessible={true}
      focusable={true}
      onClick={[Function]}
      onResponderGrant={[Function]}
      onResponderMove={[Function]}
      onResponderRelease={[Function]}
      onResponderTerminate={[Function]}
      onResponderTerminationRequest={[Function]}
      onStartShouldSetResponder={[Function]}
      style={
        Object {
          "alignItems": "center",
          "backgroundColor": "white",
          "borderRadius": 50,
          "elevation": 20,
          "height": 50,
          "justifyContent": "center",
          "opacity": 1,
          "shadowColor": "#000",
          "shadowOpacity": 1,
          "shadowRadius": 20,
          "width": 50,
        }
      }
    >
      <Text
        allowFontScaling={false}
        style={
          Array [
            Object {
              "color": "#000",
              "fontSize": 20,
            },
            undefined,
            Object {
              "fontFamily": "Material Icons",
              "fontStyle": "normal",
              "fontWeight": "normal",
            },
            Object {},
          ]
        }
      >
        
      </Text>
    </View>
    <TextInput
      allowFontScaling={true}
      onChangeText={[Function]}
      rejectResponderTermination={true}
      style={
        Object {
          "borderBottomColor": "#000",
          "borderBottomWidth": 1,
          "textAlign": "center",
          "width": 100,
        }
      }
      underlineColorAndroid="transparent"
      value=""
    />
    <View
      accessible={true}
      focusable={true}
      onClick={[Function]}
      onResponderGrant={[Function]}
      onResponderMove={[Function]}
      onResponderRelease={[Function]}
      onResponderTerminate={[Function]}
      onResponderTerminationRequest={[Function]}
      onStartShouldSetResponder={[Function]}
      style={
        Object {
          "alignItems": "center",
          "backgroundColor": "white",
          "borderRadius": 50,
          "elevation": 20,
          "height": 50,
          "justifyContent": "center",
          "opacity": 1,
          "shadowColor": "#000",
          "shadowOpacity": 1,
          "shadowRadius": 20,
          "width": 50,
        }
      }
    >
      <Text
        allowFontScaling={false}
        style={
          Array [
            Object {
              "color": "#000",
              "fontSize": 20,
            },
            undefined,
            Object {
              "fontFamily": "Material Icons",
              "fontStyle": "normal",
              "fontWeight": "normal",
            },
            Object {},
          ]
        }
      >
        
      </Text>
    </View>
  </View>
</View>
`;
